src/Generator/Generators/CSharp/CSharpTypePrinter.cs

```diff
@@ -686,6 +686,36 @@ private static string GetParameterUsage(ParameterUsage usage)
         }
 
         private CSharpExpressionPrinter expressionPrinter => new CSharpExpressionPrinter(this);
+
+        public override TypePrinterResult VisitFieldDecl(Field field)
+        {
+             
+            List<TranslationUnit> unit = new List<TranslationUnit>();
+            var cSharpSourcesDummy = new CSharpSources(Context, unit);
+            var safeIdentifier = cSharpSourcesDummy.SafeIdentifier(field.Name);
+
+            if (safeIdentifier.All(c => c.Equals('_')))
+            {
+                safeIdentifier = cSharpSourcesDummy.SafeIdentifier(field.Name);
+            }
+
+            PushMarshalKind(MarshalKind.NativeField);
+            var fieldTypePrinted = field.QualifiedType.Visit(this);
+            PopMarshalKind();
+
+            var fieldType = field.QualifiedType.Type.Desugar().IsAddress() ?
+                CSharpTypePrinter.IntPtrType : fieldTypePrinted.Type;
+
+            var fieldName = safeIdentifier;
+
+            TypePrinterResult returnTypePrinter = new TypePrinterResult();
+            if (!string.IsNullOrWhiteSpace(fieldTypePrinted.NameSuffix))
+                returnTypePrinter.NameSuffix = fieldTypePrinted.NameSuffix;
+
+            returnTypePrinter.Type = fieldType +" "+ fieldName;
```

Comment:
As a general rule, do not concatenate strings but interpolate them. There are refactoring tools, such as Refactoring Essentials and C# Essentials, which offer you to automatically convert concatenation to string.Format, and then string.Format to an interpolated string. I suggest you install both, they are extremely useful.